razorpay-subscriptions.php

```diff
@@ -53,25 +54,39 @@ class WC_Razorpay_Subscription extends WC_Razorpay
             'description',
         );
 
+        /**
+         * Contains all supported methods of woocommerce subscriptions
+         * @var array
+         */
         public $supports = array(
             'subscriptions',
             'subscription_reactivation',
             'subscription_suspension',
             'subscription_cancellation',
         );
 
+        public $settings;
```

Comment:
What are these 4 variables for?

Reply:
@captn3m0 Declaring `$subscriptions`. Removed usage of `$this->fields` and`$settings`, `$form_fields` are already declared in parent class, so removed them here.